bobtemplates/plone/viewlet/.mrbob.ini

```diff
@@ -0,0 +1,34 @@
+[questions]
+subtemplate_warning.question = Still want to run this subtemplate?
+subtemplate_warning.required = True
+subtemplate_warning.default = No
+subtemplate_warning.pre_ask_question = bobtemplates.plone.base:subtemplate_warning
+subtemplate_warning.post_ask_question = mrbob.hooks:validate_choices
+ bobtemplates.plone.base:subtemplate_warning_post_question
+subtemplate_warning.choices = Yes|No
+subtemplate_warning.choices_delimiter = |
+
+viewlet_name.question = Viewlet Name
+viewlet_name.required = True
+viewlet_name.default = training
+viewlet_name.help = Should be something like 'Training'
+
+title.question = Title
```

Comment:
I thing we can use the Viewlet Name and create Class name and title out of it.
Like it is done in most other sub-templates already.
The description, if needed somewhere, could also be autogenrated.